appveyor.yml

```diff
@@ -25,18 +25,23 @@ artifacts:
 deploy:
 - provider: NuGet
   api_key:
-    secure: 9gL9jdoH/pCMXBOG6Be6ufT7ckozGzatc1Tla3zwNsyA/ZAi8Tb+ebVLylyv6gTv
+    secure: Bt8SgvlZZ2V4hVkvaPnwBxiRy1uLw5WbksOUjMttEyramRkkooSEnF7GoPtVyZbI
   skip_symbols: true
   artifact: Release
   on:
     branch: release
 - provider: NuGet
   api_key:
-    secure: 9gL9jdoH/pCMXBOG6Be6ufT7ckozGzatc1Tla3zwNsyA/ZAi8Tb+ebVLylyv6gTv
+    secure: Bt8SgvlZZ2V4hVkvaPnwBxiRy1uLw5WbksOUjMttEyramRkkooSEnF7GoPtVyZbI
```

Comment:
This api key is already released as part of Release 2020.1. This is not a change as part of this release. The reason git shows this as a change is - Git does not compare files when it shows the changes but it actually compares the commit ids. Since we were doing squash and merge for the previous releases, it created a new commit id and the history did not have any references to actual commit ids. So, when we did a pull request, it gets all the previous commits that are not referenced in the release branch. 
For this release, we will be doing actual merge instead of 'squash merge' which should solve the problem from the next release.